libs/server/src/lib/utils/read-projects.ts

```diff
@@ -9,43 +15,31 @@ import {
 
 export function readProjects(json: any): Project[] {
   return Object.entries(json)
-    .map(([key, value]: [string, any]) => {
-      return ({
+    .map(
+      ([key, value]: [string, any]): Project => ({
         name: key,
         root: value.root,
         projectType: value.projectType,
         architect: readArchitect(key, value.architect)
-      } as any) as Project;
-    })
-    .sort(compareProjects);
-}
-
-function compareProjects(a: Project, b: Project) {
-  return a.root.localeCompare(b.root);
+      })
+    )
+    .sort((a, b) => a.root.localeCompare(b.root));
 }
 
 export function readArchitectDef(
   architectName: string,
   architectDef: any,
   project: string
 ): Architect {
-  const options = {
-    defaultValues: serializeDefaultsForConfig(architectDef.options)
-  };
-  const configurations = architectDef.configurations
+  const configurations: ArchitectConfiguration[] = architectDef.configurations
     ? Object.keys(architectDef.configurations).map(name => ({
         name,
-        defaultValues: readDefaultValues(
-          architectDef.options,
-          architectDef.configurations,
-          name
-        )
+        defaultValues: readDefaultValues(architectDef.configurations, name)
       }))
     : [];
 
   return {
-    schema: [],
-    options,
```

Comment:
Seems like the old `options` value was never used before right? So replacing it has no effect on behavior.